File: android-lib/src/androidTest/java/com/kinvey/androidTest/model/StringGenericJson.java
package com.kinvey.androidTest.model;

import com.google.api.client.json.GenericJson;
import com.google.api.client.util.Key;

/**
 * Created by yuliya on 09/21/17.
 */

public class StringGenericJson extends GenericJson {

    @Key("string")
    private String string;

    public StringGenericJson() {
    }

    public StringGenericJson(String string) {
        this.string = string;
    }

    public String getString() {
        return string;
    }

    public void setString(String string) {
        this.string = string;
    }
}

File: android-lib/src/androidTest/java/com/kinvey/androidTest/model/StringGenericJson.kt
package com.kinvey.androidTest.model

import com.google.api.client.json.GenericJson
import com.google.api.client.util.Key

/**
 * Created by yuliya on 09/21/17.
 */

data class StringGenericJson(
    @Key("string")
    var string: String? = null
): GenericJson()